cmd/tt/orgs.go

```diff
@@ -257,3 +230,35 @@ func selectOrg(label, successMsg string) *orgItem {
 	}
 	return items[index]
 }
+
+var destroyOrgsCmd = &cobra.Command{
```

Comment:
Super duper nit, but just for clarity should this be `destroyOrgCmd`? (not plural)

Reply:
I been using the naming pattern `blahRootCmd`, where the root is the main command as plural. So, `invalidateKeysCmd`, etc., since with the exception of buckets everything is aliased such that plural is the same as singular.